packages/salesforcedx-apex-replay-debugger/src/core/heapDumpService.ts

```diff
@@ -320,4 +377,19 @@ export class HeapDumpService {
   ) {
     return varContainer.name === extentValueEntry.keyDisplayValue;
   }
+
+  public isTriggerExtent(outerExtent: HeapDumpExtents) {
+    if (
+      (outerExtent.typeName === 'Boolean' ||
+        outerExtent.typeName.startsWith('List') ||
+        outerExtent.typeName.startsWith('Map')) &&
```

Comment:
Ditto.

Reply:
Commented previously. The List and Map typenames should both have the "<" as part of their comparison strings.